modal/cli/environment.py

```diff
@@ -39,10 +40,25 @@ def __rich__(self):
 def list(json: Optional[bool] = False):
     envs = environments.list_environments()
     table_data = []
+
+    env_var_default = environ.get("MODAL_ENVIRONMENT") or None
+    cfg_default = str(config.get("environment")) or None
+    active_env_count = 0
     for item in envs:
-        is_active = item.name == config.get("environment")
-        row = [item.name, item.webhook_suffix, is_active if json else RenderableBool(is_active)]
+        is_active = item.name == cfg_default or item.default is True or item.name == env_var_default
```

Comment:
Instead of setting `is_active` here and then overwriting it later, I think we should first calculate `active_env` based on the precedence rules (env var, then config, then server default), and then create the rows with `is_active = item.name == active_env`.

Reply:
Sure.

Reply:
Done